Formula/c/ca-certificates.rb

```diff
@@ -89,53 +89,103 @@ def macos_post_install
+      get_certificate_fingerprint(certificate, "/usr/bin/openssl")
     end
 
-    # Now process Mozilla certs we downloaded.
-    pem_certs_list = File.read(pkgshare/"cacert.pem")
-    pem_certs = pem_certs_list.scan(
+    # Now process Mozilla certificates we downloaded.
+    pem_certificates_list = File.read(pkgshare/"cacert.pem")
+    pem_certificates = pem_certificates_list.scan(
       /-----BEGIN CERTIFICATE-----.*?-----END CERTIFICATE-----/m,
     )
 
     # Append anything new.
-    trusted_certs += pem_certs.select do |cert|
-      fingerprint = Utils.safe_popen_write("/usr/bin/openssl", "x509", "-inform", "pem",
-                                                                       "-fingerprint",
-                                                                       "-sha256",
-                                                                       "-noout") do |openssl_io|
-        openssl_io.write(cert)
-      end
+    trusted_certificates += pem_certificates.select do |certificate|
+      fingerprint = get_certificate_fingerprint(certificate, "/usr/bin/openssl")
       fingerprints.add?(fingerprint)
     end
 
     pkgetc.mkpath
-    (pkgetc/"cert.pem").atomic_write(trusted_certs.join("\n") << "\n")
+    (pkgetc/"cert.pem").atomic_write(trusted_certificates.join("\n") << "\n")
+  end
+
+  def get_certificate_fingerprint(certificate, openssl_binary = "openssl")
+    Utils.safe_popen_write(openssl_binary, "x509", "-inform", "pem",
+                                                   "-fingerprint",
+                                                   "-sha256",
+                                                   "-noout") do |openssl_io|
+      openssl_io.write(certificate)
+    end
+  end
+
+  def load_certificates_from_file(file_path, trusted_certificates, fingerprints, certificate_type)
+    file_path = Pathname(file_path)
+    return unless file_path.exist?
+
+    certificates_list = file_path.read
+    certificates = certificates_list.scan(
+      /-----BEGIN CERTIFICATE-----.*?-----END CERTIFICATE-----/m,
+    )
+    loaded_count = 0
+    certificates.each do |certificate|
+      fingerprint = get_certificate_fingerprint(certificate)
+      if fingerprints.add?(fingerprint)
+        trusted_certificates << certificate
+        loaded_count += 1
+      end
+    rescue ErrorDuringExecution
+      # Skip invalid certificate
+      next
+    end
+    puts "Loaded #{loaded_count} #{certificate_type} certificates" if loaded_count.positive?
   end
 
   def linux_post_install
     rm(pkgetc/"cert.pem") if (pkgetc/"cert.pem").exist?
     pkgetc.mkpath
-    cp pkgshare/"cacert.pem", pkgetc/"cert.pem"
+    # Integrate system certificates if OpenSSL is available
+    unless which("openssl")
+      opoo "Cannot find OpenSSL: skipping system certificates."
+      puts "To include custom system certificates run:"
+      puts <<~EOS
+        brew install openssl
+        brew postinstall ca-certificates
```

Comment:
We usually indent shell commands in these messages. To do that here:
```suggestion
      puts <<~EOS
        To include custom system certificates run:
          brew install openssl
          brew postinstall ca-certificates
```